client/src/receiver/index.ts

```diff
@@ -26,7 +26,11 @@ fetch(`/api/stream/${humanId}`)
     if (res.status === 200) {
       return res.json()
     }
-    const playbackUrl = `https://cdn.livepeer.com/hls/${humanId}/index.m3u8`
+    const playbackUrl = humanId.startsWith('https://')
```

Comment:
Can the server API have a single concept of `humanId`?

Reply:
Just reverted [that commit](https://github.com/victorges/justcast.it/pull/29/commits/b791f08817e1bb9c07eea517d857729ce65e18f4), it was misplaced on this PR.